### ISSUE TITLE
[Bug] Unable to insert configuration into sublime-project file

### ISSUE THREAD

**Issue by LDAP** [daveleroy/sublime_debugger#189]
Debugger fails to add a configuration to a sublime project if the key `debugger_configurations` is not present.

```
Debugger: ProjectConfiguration.reload
Debugger: No project associated with window
Debugger:Traceback (most recent call last):
  File "/home/lucas/.config/sublime-text/Packages/Debugger/modules/adapters_registry.py", line 81, in _insert_snippet
    raise core.Error('Unable to find debugger_configurations')
Debugger.modules.core.error.Error: Unable to find debugger_configurations

--
error: Unable to insert configuration into sublime-project file: Copied to clipboard instead
```

**Comment by daveleroy** [comment]
Snippet insertion is not supported when not using a sublime-project at the moment

**Comment by LDAP** [comment]
That was with an active sublime project.

**Comment by dongguaguaguagua** [comment]
change your sublime-project into this:
```json
{
	"folders": [
		{
			"path": "."
		},
	],
	"debugger_tasks": [
	],
	"debugger_configurations": [
	],
}

```

**Comment by LDAP** [comment]
> change your sublime-project into this:

Thanks @dongguaguaguagua I already did this as a workaround. However, Debugger should be able to insert snippets into sublime projects with missing `debugger_*` keys as well.

**Comment by daveleroy** [comment]
Its supposed to add `debugger_configurations` when first opening the debugger. Should be fixed here 73acaf1b139c0f44098173decb8047c69cd3aebd
